### ISSUE TITLE
invalid handle while importing block entity

### ISSUE THREAD

**Issue by songyuxiang** [mozman/ezdxf#65]
source_dwg = ezdxf.readfile(in_file1)
target = ezdxf.new(source_dwg .dxfversion)
importer = ezdxf.Importer(source_dwg, target_dwg)
importer.import_tables()
importer.import_modelspace_entities()
target .saveas(out_file)

I got errors:
Traceback (most recent call last):
  File "E:/reseaux/example.py", line 49, in <module>
    get_reseaux()
  File "E:/reseaux/example.py", line 46, in get_reseaux
    importer.import_modelspace_entities()
  File "C:\Users\y.song\AppData\Local\Programs\Python\Python36\lib\site-packages\ezdxf\tools\importer.py", line 52, in import_modelspace_entities
    target_modelspace.add_entity(new_entity)  # add entity to modelspace
  File "C:\Users\y.song\AppData\Local\Programs\Python\Python36\lib\site-packages\ezdxf\legacy\layouts.py", line 124, in add_entity
    for linked_entity in entity.linked_entities():
  File "C:\Users\y.song\AppData\Local\Programs\Python\Python36\lib\site-packages\ezdxf\dxfentity.py", line 136, in linked_entities
    entity = wrap(link)
  File "C:\Users\y.song\AppData\Local\Programs\Python\Python36\lib\site-packages\ezdxf\legacy\factory.py", line 102, in wrap_handle
    raise DXFKeyError('invalid handle #{}'.format(handle))
ezdxf.lldxf.const.DXFKeyError: 'invalid handle #287B'
![capture](https://user-images.githubusercontent.com/24799927/51379261-4b1a8780-1b0f-11e9-83a1-d9a023ed948b.PNG)

**Comment by mozman** [comment]
1. It is not possible to fix anything without the used data!

2. WHY are you importing the data into a new drawing, just open the source drawing and save  it as a new file.

3. For the upcoming release 0.9.0: moved `Importer()` from `ezdxf.tools` to `ezdxf.addons` - internal structures of modern DXF files are too complex and too undocumented to support importing data in a reliable way - using Importer() may corrupt your DXF files or just don't work!

Don't relay on this feature, it may be removed in the future.

**Comment by songyuxiang** [comment]
Thanks for your reply.
I used just a simplest case to take a try. I was trying to merge two dxf files(add entities for one file to another).

_source_dwg = ezdxf.readfile("source.dxf")
target_dwg = ezdxf.readfile("target.dxf")
for e in source_dwg.entities:
    source_dwg.modelspace().unlink_entity(e)
    target_dwg.modelspace().add_entity(e)
target_dwg.saveas("merge.dxf")_

Here is my code. But I got a handle key error.
I checked the entitydb dictionary, and there is no new handle key added when add_entity().
I would like to know how to do this.

Best regards

[merge.zip](https://github.com/mozman/ezdxf/files/2778556/merge.zip)
